scripts/check_links.py

```diff
@@ -0,0 +1,48 @@
+#! /usr/bin/env python
+"""
+Goes through all the inline-links in markdown files and reports the breakages.
+"""
+import re
+import sys
+import pathlib
+from multiprocessing.dummy import Pool
+from collections import namedtuple
+from typing import Tuple
+import requests
+
+MatchTuple = namedtuple('MatchTuple', 'source name link')
```

Comment:
Did you know that you can create `NamedTuple` classes by inheriting from `typing.NamedTuple`? It allows you to have types in the tuple, which is nice - see [here](https://github.com/allenai/allennlp/blob/master/allennlp/models/constituency_parser.py#L21) for an example.

Reply:
Cool, I didn't know. Let me check ...